docs/pt/docs/help-fastapi.md

```diff
@@ -0,0 +1,148 @@
+# Ajuda FastAPI - Obter Ajuda
+
+Você gosta do **FastAPI**?
+
+Você gostaria de ajudar o FastAPI, outros usários, e o autor?
+
+Ou você gostaria de obeter ajuda relacionada ao **FastAPI**?
```

Comment:
Ou você gostaria de obter ajuda relacionada ao **FastAPI**?

Reply:
Hi @peidrao thanks for your review. Could you help me understand what is the problem here? Your review is the exaclty same phrase that the one i used.